app/client/src/modules/ui-builder/ui/wds/WDSCustomWidget/component/index.tsx

```
@@ -13,198 +1,187 @@
+          model: model,
+        });
+      }
+    },
+    [model],
+  );
+
+  useEffect(
+    // similar to model change, iframe can listen to changes to theme with
+    // `appsmith.onThemeChange` function.
+    function handleThemeUpdate() {
+      if (iframe.current && iframe.current.contentWindow && isIframeReady) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)